tasks/audit-cat2.yml

```diff
@@ -0,0 +1,80 @@
+- name: "MEDIUM | RHEL-07-040640 | AUDIT | The SSH public host key files must have mode 0644 or less permissive."
+  find:
+      paths: /
+      recurse: yes
+      file_type: file
+      patterns: '*.pub'
+      hidden: true
+  failed_when: no
+  changed_when: no
+  ignore_errors: yes
+  register: rhel_07_040640_audit
+  tags:
+      - cat2
+      - high
+      - audit
+      - RHEL-07-040640
+      - always
+      - ssh
+
+- name: "MEDIUM | RHEL-07-040650 | AUDIT | The SSH private host key files must have mode 0600 or less permissive."
+  find:
+      paths: /
+      recurse: yes
+      file_type: file
+      patterns: '*ssh_host*key'
+      hidden: true
+  failed_when: no
+  changed_when: no
+  ignore_errors: yes
+  register: rhel_07_040650_audit
+  tags:
+      - cat2
+      - high
+      - audit
+      - RHEL-07-040650
+      - always
+      - ssh
+
+- name: "MEDIUM | RHEL-07-040230 | AUDIT | The operating system, if using PKI-based authentication, must implement a local cache of revocation data to certificate validation in case of the inability to access revocation information via the network."
+  stat:
+      path: /var/lib/pki-kra/conf/server.xml
+  register: rhel_07_040230_audit
```

Comment:
Please add `failed_when`, `changed_when`, and `ignore_errors` to all audit tasks.